doc/faq.rst

```diff
@@ -3,6 +3,19 @@ FAQ
 
 This is the list of Frequently Asked Questions for restic.
 
+Can restic resume an interrupted backup?
+----------------------------------------
+
+Yes it can. When backing up, restic periodically writes index files to keep a record of the uploaded
+data. Even if there's no snapshot created in the end (due to the backup being interrupted), these
+indexes are stored in the database for the data that has been uploaded so far. Next time restic runs,
+it is then able to find the uploaded data through these indexes, and thereby reference it again without
+having to upload it a second time. This effectively makes it continue from where it saved the last index.
+
+It does not matter if the backup was interrupted by the user (e.g. for having to pause it temporarily)
```

Comment:
I'd leave out the parenthesized part. The user's intention is irrelevant.

Reply:
You're right, less is more.